src/sharding/shard.ts

```diff
@@ -17,8 +19,8 @@ const sharder = new ShardingManager('./bot/index.js', {
 
 sharder.spawn().then(shards => {
   shards.forEach(shard => {
-    shard.on('ready', () => logger.log(`Shard ${shard.id} has been spawned.`, 'SHARDS'));
-    shard.on('disconnect', () => logger.error(`Shard ${shard.id} disconnected from Discord Websocket.`, 'SHARDS'));
-    shard.on('death', () => logger.error(`Shard ${shard.id} died! Respawning...`, 'SHARDS'));
+    shard.on('ready', () => logger.debug(`Shard ${shard.id} has been spawned.`));
+    shard.on('disconnect', () => logger.error(`Shard ${shard.id} disconnected from Discord Websocket.`));
+    shard.on('death', () => logger.error(`Shard ${shard.id} died! Respawning...`));
```

Comment:
TypeScript is throwing an error here
```suggestion
    shard.on('ready', () => {
      logger.debug(`Shard ${shard.id} has been spawned.`)
    });
    shard.on('disconnect', () => {
      logger.error(`Shard ${shard.id} disconnected from Discord Websocket.`)
    });
    shard.on('death', () => {
      logger.error(`Shard ${shard.id} died! Respawning...`)
    });
```